src/bin/resource_tracing.rs

```diff
@@ -549,10 +549,16 @@ fn extract_pickup_location<'r>(
         }
     } else {
         // Phazon suit override
-        assert_eq!(pickup.kind, 23);
-        ScriptObjectLocation {
-            layer: 1,
-            instance_id: 68813644,
+        if pickup.kind ==  23 { // phazon suit
+            ScriptObjectLocation {
+                layer: 1,
+                instance_id: 68813644,
+            }
+        } else {
+            ScriptObjectLocation {
+                layer: 0,
+                instance_id: 0xFFFFFFFF,
+            }
```

Comment:
Just curious, why this new condition?

Could this be an `else-if` chain instead?